docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md

```diff
@@ -258,6 +261,62 @@ One major benefit that we get from doing this is that we know the building of th
 
 Since these images will contain code from within a private repo, we need to make sure that unauthenticated users do not have access to these images. We prevent this by adding an [RBAC file](https://github.com/openshift/release/pull/36522/files#diff-e129896e392bc9ced499cb19ddf549b361784048825b25670755fc432f87f542) to only allow authenticated users to pull from the target registry namespace.
 
+### Mirror Quay Images
+
+There might be a case when we need to use the existing quay image in our scenario. The image can be in either public or private registry. For this purpose, Openshift CI allows to use external images by mirroring them into Openshift CI registry. Create separate pull request to mirror image in Openshift CI and get it merged before using that image in any job or pull request rehearsals. Once the image is mirrored in Openshift CI registry, use it as `base_image` in config file.
+
+#### How to Mirror Public Image
+
+If the image is public, we can mirror the image by adding a mapping in file `core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci` of [openshift/release](https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci) repository like this:
+
+```yaml
+quay.io/QUAY_REPOSITORY/test-image:latest registry.ci.openshift.org/ci/test-image:latest
+```
+
+Example: For AMQ Streams scenario, container image for tests is in public quay repository. In order to run tests using this image, pull request [Mirror Strimzi/AMQ Streams test images to CI registries](https://github.com/openshift/release/pull/37512) is created.
+
+#### How to Mirror Private Image
+
+If image is in private registry, following are the steps required to mirror that image.
+
+1. Create a new sub folder in `core-services/image-mirroring` folder of [openshift/release](https://github.com/openshift/release/tree/master/core-services/image-mirroring) repository with a name something similar to your scenario name. 
+
+   Example: For 3scale scenario, folder name is `3scale-qe`.
+
+2. Add `OWNERS` file and a file with name similar to `mapping_$FOLDER_NAME-*` for adding image mapping.
+
+    ```yaml
+    quay.io/QUAY_REPOSITORY/IMAGE_NAME:latest registry.ci.openshift.org/$FOLDER_NAME/IMAGE_NAME:latest
```

Comment:
This is the format of image mapping, right?
Could you elaborate the example below, by adding the image mapping for 3scale scenario and then the 3scale file name, to avoid confusion
Or
If you already add the format and example of image mapping before this, you can skip adding that format & example image mapping

Reply:
same comment as previous one. For this readme, I meant to have the mapping format and example together.
If there is an example for mapping provided its fine.

Reply:
This section is removed, marking as resolved!